src/App/Models/WindowsEzStartup.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace App.Models
+{
+    public class WindowsEzStartup : IEzStartup
```

Comment:
no change now, I'll submit a PR later. But we're going to make this an internal method to FORCE people to use the interface and DI framework. 

I'll add this boilerplate after you complete this PR